### ISSUE TITLE
Run and Gun and Close Encounters can proc on the same turn with Teamwork

### ISSUE THREAD

**Issue by allenhu0912** [long-war-2/lwotc#618]
1. Run and Gun
2. Perform action of your choice
3. Teamwork the assault for another action
4. Shoot within range of CE to get another action back

**Comment by pledbrook** [comment]
Did this not work with LW2 and Command?

**Comment by noirforest** [comment]
Yes, this trick work on vanilla LW2 too. Command is a tactics move per mission, IMO this trick could be fine.

**Comment by pledbrook** [comment]
By performing the action of your choice, you consume the Run And Gun action point, which frees up Close Encounters to work when you use Teamwork or Command.

**Comment by allenhu0912** [comment]
Yeah, but tooltips do say than RnG/CE don't work with each other. Perhaps clarify?

**Comment by Favid** [comment]
Peter is right that, as the code is written, they should be allowed. However, the descriptions of Hit and Run and Close Encounters explicitly say "Cannot be used on the same turn as Run and Gun." and "Close Encounters cannot be used on the same turn as Run and Gun.", respectively. Because the descriptions explicitly state that they cannot be used in the same **turn** as R&G, there's an argument that can be made where Command letting this be bypassed in LW2 could be considered a bug that was never fixed, which would then mean that the same case with Teamwork would also be a bug.

Ultimately the decision is up to Peter. If the current functionality is deemed ok, then the descriptions for H&R and CE should be updated.

**Comment by Chaotic-Entropy** [comment]
Perhaps "Cannot be used in the same action as Run and Gun" or something.
